packages/syft/src/syft/protocol/protocol_version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "1": {
+  "dev": {
```

Comment:
I think the procotol_version.json seems incorrect, maybe to incorrect merge with dev. Can you reset this file with `dev` 
`git checkout dev -- packages/syft/src/syft/protocol/protocol_version.json` and then in a new terminal
`tox -e syft.protocol.check` to stage any protocol changes.

Reply:
I noticed there are some differences in my branch's `protocol_version.json` and `dev`'s `protocol_version.json`. Will simply copying the content of `dev`'s `protocol_version.json` into mine work?